src/apps/material/material.dev.tsx

```diff
@@ -12,28 +12,33 @@ export default {
       control: { type: "text" }
     },
     materialHeaderAuthorByText: {
-      name: "Af forfatter",
+      name: "By (author)",
       defaultValue: "Af ",
       control: { type: "text" }
     },
     periodikumSelectYearText: {
-      name: "År",
+      name: "Year",
       defaultValue: "År",
       control: { type: "text" }
     },
     periodikumSelectWeekText: {
-      name: "Uge",
+      name: "Week",
       defaultValue: "Uge",
       control: { type: "text" }
     },
     reserveBookText: {
-      name: "Reserve book",
+      name: "Reserve",
       defaultValue: "RESERVER BOG",
       control: { type: "text" }
     },
-    fineOnBookshelfText: {
-      name: "Fine on bookshelf",
-      defaultValue: "FINE PÅ HYLDEN",
+    findOnBookshelfText: {
+      name: "Find on bookshelf",
+      defaultValue: "FIND PÅ HYLDEN",
```

Comment:
Consider if we should capitalize the text using CSS instead of static text.

Note that this outside the scope of this PR and also applies to other buttons.

Reply:
You're right. Very good suggestion!. 
I have added a utility class that does this in the other PR I have open in the design system.